cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java

```diff
@@ -166,51 +166,51 @@ public void execute() {
         if (!this.buildAll && (this.argList == null || this.argList.size() == 0)) {
             CommandUtil.printError(this.errStream,
-                    "'build' command requires a module name or a Ballerina file to build/compile. use '-a' or " +
-                    "'--all' flag to build/compile all the modules of the project.",
+                    "'build' command requires a module name or a Ballerina file to build/compile. Use '-a' or " +
+                    "'--all' to build/compile all the modules of the project.",
                     "ballerina build {<ballerina-file> | <module-name> | -a | --all}",
                     false);
             CommandUtil.exitError(this.exitWhenFinish);
             return;
         }
         
-        // validation and decide source root and source full path
+        // Validate and decide the source root and the full path of the source.
         this.sourceRootPath = null != this.sourceRoot ?
                 Paths.get(this.sourceRoot).toAbsolutePath() : this.sourceRootPath;
         Path sourcePath = null;
         Path targetPath;
         
-        // when -a or --all flag is provided. check if the command is executed within a ballerina project. update source
-        // root path if command executed inside a project.
+        // When -a or --all is provided, check if the command is executed within a Ballerina project. Update source
+        // root path if the command is executed inside a project.
         if (this.buildAll) {
-            // when building all the modules
-            //// check if output flag is set
+            // Check if the output flag is set
+            //// when building all the modules.
             if (null != this.output) {
                 CommandUtil.printError(this.errStream,
-                                       "'-o' and '--output' flag is only supported for building a single Ballerina " +
+                                       "'-o' and '--output' are only supported for building a single Ballerina " +
                                                "file.",
                                        "ballerina build -o <output-file> <ballerina-file> ",
                                        true);
                 CommandUtil.exitError(this.exitWhenFinish);
                 return;
             }
         
-            //// validate and set source root path
+            //// Validate and set the path of the source root.
             if (!ProjectDirs.isProject(this.sourceRootPath)) {
                 Path findRoot = ProjectDirs.findProjectRoot(this.sourceRootPath);
                 if (null == findRoot) {
                     CommandUtil.printError(this.errStream,
-                                           "you are trying to build/compile a Ballerina project but there is no " +
+                                           "You are trying to build/compile a Ballerina project that does not have a " +
```

Comment:
This should be "you", going by the convention we use

Reply:
fixed.